Tests/test_AlignIO_FastaIO_unittest.py

```diff
@@ -0,0 +1,255 @@
+from Bio._py3k import StringIO
+from Bio.AlignIO import FastaIO
+import os
+import unittest
+
+simple_example = \
```

Comment:
You've lost potentially important white space information compared to the string as used in ``Bio/AlignIO/FastaIO.py``